_episodes_rmd/07-control-flow.Rmd

```diff
@@ -390,7 +390,7 @@ output_vector2
 > >  upperThreshold <- 70
 > >  
 > > for( iCountry in unique(gapminder$country) ){
-> >     tmp <- mean(subset(gapminder, country==iCountry)$lifeExp)
+> >     tmp <- mean(gapminder[gapminder$continent == iContinent, "lifeExp"])
```

Comment:
Isn't this going to be an issue since the loop is iterating over countries not continents?

Reply:
Good catch!